static/src/stylesheets/module/commercial/_creatives.scss

```diff
@@ -1,5 +1,7 @@
 .creative__tracking-pixel {
     position: absolute;
+    width: 1px;
```

Comment:
I think we are sacrificing readability for a micro-optimization.

Are there figures comparing the two approaches?